packages/core/src/css/cssConfig.ts

```diff
@@ -0,0 +1,145 @@
+import { createRequire } from 'node:module';
+import path from 'node:path';
+import type { RsbuildConfig, RsbuildPlugin } from '@rsbuild/core';
+import { CSS_EXTENSIONS_PATTERN } from '../constant';
+import { RemoveCssExtractAssetPlugin } from './RemoveCssExtractAssetPlugin';
+
+const require = createRequire(import.meta.url);
+
+export const RSLIB_TEMP_CSS_DIR = '__rslib_css__';
+
+// https://rsbuild.dev/zh/config/output/css-modules#cssmodulesauto
+export type CssLoaderOptionsAuto =
```

Comment:
`CSSLoaderModulesOptions` could be imported from `@rsbuild/core` https://github.com/web-infra-dev/rsbuild/blob/50a92951462aa7b6f95c5ab548f3a96305e0de62/packages/core/src/types/thirdParty.ts#L68-L76

Reply:
done by imported in @rsbuild/core